### PR TITLE
fix: update telemetry for app studio plugin

### PR DESCRIPTION
**Pull request by nliu-ms**
- update telemetry for app studio plugin

### CODE REVIEW

File: packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts [hunk 1]
```diff
@@ -10,11 +10,14 @@ export enum TelemetryPropertyKey {
   errorCode = "error-code",
   errorMessage = "error-message",
   validationResult = "validation-result",
+  success = "success",
 }
 
 enum TelemetryPropertyValue {
   UserError = "user",
   SystemError = "system",
+  success = "yes",
```
Comment (dilin-MS) [inline packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts:19]: Why the value is not `success` and `fail`?

Comment (nliu-ms) [reply packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts:21]: It's aligned with other plugin telemetries

Comment (chagong) [reply packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts:21]: @dilin-MS The property name is "success", it's like asking if the operation is succeeded, so the answer should be "yes" or "no".

File: packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts [hunk 2]
```diff
@@ -39,6 +44,9 @@ export class TelemetryUtils {
       properties = {};
     }
     properties[TelemetryPropertyKey.component] = Constants.PLUGIN_NAME;
+    if (this.ctx.app.id) {
```
Comment (chagong) [inline packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts:47]: ctx.app.id is not "remote teams app id", it the id in manifest file, and its value could be a placeholder like "{appid}". we opened a bug for that before. So please make sure you get appid from env config file.

**Review by chagong — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

File: packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts (hunk 1 above)
Comment (dilin-MS) [reply packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts:23]: What about `isSuccessful` as property name and `true` / `false` as possible value so as to increase code readability?

**Pull request merged**